diesel_compile_tests/rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2018-11-27
+nightly-2019-02-26
```

Comment:
That changes the output of our compile tests, which means you need to go though that errors and adjust them. (There is a script for that)